tests/testthat/test-template.R

```diff
@@ -778,3 +778,249 @@ test_that("curve_metric_summarizer() deals with characters in truth", {
 
   expect_identical(roc_curve_res, roc_curve_exp)
 })
+
+## dynamic_survival_metric_summarizer -----------------------------------------
+
+# To be removed once brier_survival() is added
```

Comment:
testing `dynamic_survival_metric_summarizer()` is a bit of a chicken and egg situation with the survival metrics.
So i added this dummy `brier_survival_vec()` that does enough to allow testing for `dynamic_survival_metric_summarizer()`. And it will be removed in the next PR